### PR TITLE
Adding os-client-config

### PR DESCRIPTION
**Pull request by brainstorm**
Breaking up dependencies for PR #3421 (OpenStack+Ansible)

### CODE REVIEW

File: recipes/os-client-config/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "os-client-config" %}
+{% set version = "1.27.0" %}
```
Comment (isuruf) [inline recipes/os-client-config/meta.yaml]: Shouldn't this be `1.28.0`?

File: recipes/os-client-config/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - appdirs
+    - pyyaml
+
+test:
+  imports:
+    - os_client_config
+
+about:
+  home: https://docs.openstack.org/developer/os-client-config/
+  license: Apache 2.0
```
Comment (isuruf) [inline recipes/os-client-config/meta.yaml]: Add `license_file: LICENSE` here

**Comment by isuruf** [comment]
Already available at https://github.com/conda-forge/os-client-config-feedstock

**Comment by jakirkham** [comment]
Thanks @isuruf.

Also would encourage you, @brainstorm, to add yourself as a maintainer in the feedstock if you are interested. 😉

**Pull request closed without merging**